src/kartograf/atom_mapper.py

```diff
@@ -101,12 +103,15 @@ def __init__(
         _mapping_algorithm : str, optional
             mapping_algorithm.linear_sum_assignment - this allows to swap the
             optimization algorithm. Not recommended.
+        allow_partial_fused_rings: bool
+            If we should allow partially mapped fused rings `True` or not `False`, default `True`.
```

Comment:
Should this be `True` or `False`?

Reply:
Ah I was trying to make it clear what each option did but maybe I should change it to 
`If we should allow partially mapped fused rings (True) or not (False)`?

Reply:
The () would be useful visually but not a massive issue. In case Hannah is asking if it should default to False instead - for now we have to leave it to True, but we can deprecate the behaviour and aim for switching to False in the near future.